Makefile

```diff
@@ -27,7 +27,7 @@ migrate_test: ## Run database migrations for test.
 	hack/migrate.sh postgres
 
 test: ## Run tests.
-	go test -p 1 -v $(CHECK_FILES)
+	go test ./... -p 1 -race -v $(CHECK_FILES) -count=1 -coverpkg ./api/... -coverprofile=coverage.out
```

Comment:
`-coverpkg ./api/...`
Just FYI how -coverpkg works:
```
-coverpkg pattern1,pattern2,pattern3
	    Apply coverage analysis in each test to packages matching the patterns.
	    The default is for each test to analyze only the package being tested.
```
so for your case I think it is reasonable to have `-coverpkg ./...` but maybe I am wrong

Reply:
So if your test will cover some code in other packages it won't be counted 

And (not sure here) the total coverage won't track other packages which may be not what you desire